app/models/assessment.rb

```diff
@@ -46,7 +46,8 @@ class Assessment < ApplicationRecord
   # Constants
   ORDERING = "due_at ASC, name ASC".freeze
   RELEASED = "start_at < ?".freeze
-
+  VALID_NAME_REGEX = /^[A-Za-z][A-Za-z0-9_-]*$/.freeze
```

Comment:
Ruby identifiers can start with underscores, should we allow that?

Reply:
I left out underscores because in the current regex that would allow for all underscores to be valid. Essentially, it makes the validation more complicated so rather not allow it.